src/libraries/System.Collections/src/System/Collections/BitArray.cs

```diff
@@ -120,9 +120,11 @@ public BitArray(byte[] bytes)
             _version = 0;
         }
 
+#if !NET6_0_OR_GREATER
```

Comment:
Why are these ifdefs needed... isn't this file only compiled for netcoreappcurrent?

Reply:
> isn't this file only compiled for netcoreappcurrent?

This is the point I don't know (alluded in the description).
But if so, I'll just remove them.

My point is just to make sure that .NET Core 3.1 doesn't regress, as it's a LTS through Dec, 2022.

Reply:
They're not needed.  This file is only included in System.Collections.csproj, which only builds for netcoreappcurrent:
https://github.com/dotnet/runtime/blob/bde0322a9e11fba40a7a65c6d38200d6514efc72/src/libraries/System.Collections/src/System.Collections.csproj#L5

Base64Decoder/Encoder.cs are only included in System.Memory, which only builds for netcoreappcurrent:
https://github.com/dotnet/runtime/blob/bde0322a9e11fba40a7a65c6d38200d6514efc72/src/libraries/System.Memory/src/System.Memory.csproj#L6

Reply:
Thanks! 👍🏻